enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/replication/tx/ReplicatedTransactionCommitProcess.java

```diff
@@ -111,15 +114,13 @@ transaction content does eventually get replicated (e.g. delayed on the network)
             }
             catch ( InterruptedException | TimeoutException  e )
```

Comment:
I don't think we are giving up when interrupting, rather we're retrying. Are you suggesting we have different behaviour depending on whether there's a timeout or interrupt?